felipebaltazar/Maui.TutorialCoachMark
# Project analyzers

### Overview
Analyzer project has been created

### Issues Resolved

- fixes #5 

### API Changes
None

### Platforms Affected
- All

### Behavioral Changes

Now the analyzers will check if `UseTutorialCoachMark` was called


### Testing Procedure
Try to install Maui.TutorialCoachMark package ant compile it without include `UseTutorialCoachMark`
We will see an error

### PR Checklist ###
